mediabridge-frontend/src/pages/SelectMovies.tsx

```diff
@@ -11,21 +12,31 @@ const SelectMovies = ({
   addMovie: (movie: Movie) => void;
   removeMovie: (id: string) => void;
 }) => {
+  const [recommendations, setRecommendations] = useState<string[]>([]);
+
   return (
     <div className="min-h-screen flex flex-col">
       {/* Top full-width section */}
       <div className="w-screen">
         <div className="max-w-screen-mx flex justify-center">
-          <MovieSearch movies={movies} addMovie={addMovie} />
+          <MovieSearch movies={movies} addMovie={addMovie} setRecommendations={setRecommendations} />
         </div>
       </div>
 
       {/* Content section below */}
-      <div className="px-4 py-8">
+      {
+        recommendations.length > 0 ?
```

Comment:
Maybe you can just use MovieItem here? Or MovieList without the `removeMovie` property? Then the `MovieList` component could check if the optional `removeMovie` function was passed in, and if not, hide the remove buttons.